.github/workflows/test.yml

```diff
@@ -117,6 +117,7 @@ jobs:
           REDIS_HOST: 127.0.0.1
 
   test:
+    if: ${{ always() }}
     runs-on: ubuntu-latest
     needs:
       - test-unit
```

Comment:
Do we still need this then?

Reply:
Its the pattern which is used by the probot/octokit team